app/config/index.js

```diff
@@ -33,7 +33,7 @@ value.isProd = value.env === 'production'
 value.processingSubscription = mqConfig.processingSubscription
 value.submitSubscription = mqConfig.submitSubscription
 value.returnSubscription = mqConfig.returnSubscription
-value.statementTopic = mqConfig.statementTopic
```

Comment:
The `statementTopic` needs to remain as is untouched by this ticket.

That is still the mechanism by which statement will leave this service.  We'll write them to `ffc-pay-statements` topic.

This ticket covers a new subscription to topic `ffc-pay-statement-data` through subscription `ffc-pay-statement-constructor`.

So needs a new line adding, say `statementDataSubscription` or similar

Reply:
had another go, for some reason I got it into my head the `statementTopic` was just a placeholder that needed to be updated. oops.